cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -1,50 +1,65 @@
-                    <input type="checkbox" id="id_timed_examination" name="timed_examination" class="input input-checkbox" />
-                    <label for="id_timed_examination" class="label">
-                      <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
-                      <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
-                      <%- gettext('This exam is timed') %>
-                    </label>
+                <li class="field-radio">
+                    <input type="radio" id="id_not_timed" name="proctored" class="input input-radio" checked="checked"/>
+                    <label for="id_not_timed" class="label">
+                      <%- gettext('None') %>
+                    </label>  
                 </li>
             </ul>
         </div>
+
         <div class='exam-time-list-fields'>
+            <ul class="list-fields list-input">
+                <li class="field-radio">
+                    <input type="radio" id="id_timed_exam" name="proctored" class="input input-radio" />
+                    <label for="id_timed_exam" class="label">
+                      <%- gettext('Timed') %>
+                    </label>
+                </li>
+                <p class='field-message'> <%- gettext('Use a timed exam to limit the time learners can spend on problems in this subsection. Learners must submit answers before the time expires. You can allow additional time on per learner basis through the Instructor Dashboard.') %> </p>
+
+            </ul>
+        </div>
+        <% if (enable_proctored_exam) { %>
+            <div class='exam-time-list-fields'>
+                <ul class="list-fields list-input">
+                    <li class="field-radio">
+                        <input type="radio" id="id_proctored_exam" name="proctored" class="input input-radio" />
+                        <label for="id_proctored_exam" class="label">
+                            <%- gettext('Proctored') %>
+                        </label>
+                    </li>
+                    <p class='field-message'> <%- gettext('Proctored exams are timed, and software records video of each learner taking the exam. These videos are then reviewed by a third party.') %> </p>
+                </ul>
+            </div>
+            <div class='exam-time-list-fields'>
+                <ul class="list-fields list-input">
+                    <li class="field-radio">
+                        <input type="radio" id="id_practice_exam" name="proctored" class="input input-radio" />
+                        <label for="id_practice_exam" class="label">
+                          <%- gettext('Practice Proctored') %>
+                        </label>
+                    </li>
+                    <p class='field-message'> <%- gettext("Use a practice proctored exam to introduce learners to the proctoring tools and processes. Results of the practice exam do not count towards the learner's grade.") %> </p>
```

Comment:
Add a unique `id` here, then add `aria-describedby="[said id]"` to the `input` it's associated with above.